frontend/search/searchController.js

```diff
@@ -36,18 +36,22 @@
             searchCtrl.institutions = [];
         };
 
-        searchCtrl.search = function search() {
+        searchCtrl.search = function search(ev) {
             if (searchCtrl.search_keyword) {
-                searchCtrl.makeSearch(searchCtrl.search_keyword, 'institution');
+                searchCtrl.makeSearch(searchCtrl.search_keyword, 'institution').then(() => {
+                    if (Utils.isMobileScreen()) {
+                        searchCtrl.showSearchFromMobile(ev);
+                    }
+                });
                 refreshPreviousKeyword();
```

Comment:
is there any problem with this function being "parallel" with `makeSearch`?

Reply:
you mean put it inside the then block?